Week-6/Ex5.js

```diff
@@ -0,0 +1,19 @@
+
+//SC: O(1)
+//TC: O(n^2)
+
+function isPairWithGivenDiffPresent(numbers, diff) {
+    let isPresent = false;
+    for(let n of numbers) {
+        let currentDiff = n - diff;
```

Comment:
This logic might not work good if the diff is considered as | x1-x2|.

Edge cases again.